### DISCUSSION TITLE
Starting sliver-client and sliver-server in kali

### DISCUSSION THREAD

**Discussion by mdeyn** [BishopFox/sliver#1677] [Q&A]
Hi all,

Getting started with sliver has been great so far, git cloned with SSH and then onto make.

I encounter a problem when trying to start both ./sliver-client and ./sliver-server. Please see screenshot below. Both seem to disappear when I try and run. Don't think this is a path issue as /usr/local/bin is already there.

![kali_issue_2024-05-08 16_34_00-](https://github.com/BishopFox/sliver/assets/100421664/d609d3e9-4358-4d87-a7a5-df93a55d5065)

Any pointers much appreciated

**Comment by moloch--** [comment]
Honestly I have no idea what's going on here, but it's almost certainly an issue with the machine not Sliver.
